packages/@aws-cdk/aws-glue-alpha/lib/job.ts

```diff
@@ -702,11 +711,13 @@ export class Job extends JobBase {
     this.sparkUILoggingLocation = sparkUI?.location;
     const continuousLoggingArgs = props.continuousLogging?.enabled ? this.setupContinuousLogging(this.role, props.continuousLogging) : {};
     const profilingMetricsArgs = props.enableProfilingMetrics ? { '--enable-metrics': '' } : {};
+    const observabilityMetricsArgs = props.enableObservabilityMetrics ? { '--enable-observability-metrics': '' } : {};
```

Comment:
According to the doc, does the value have to be `'true'` instead of `''`?

> To enable this feature, set the value of this parameter to true.

https://docs.aws.amazon.com/glue/latest/dg/aws-glue-programming-etl-glue-arguments.html

```suggestion
    const observabilityMetricsArgs = props.enableObservabilityMetrics ? { '--enable-observability-metrics': 'true' } : {};
```